zome-functions-transform.js

```diff
@@ -0,0 +1,73 @@
+const fs = require('fs')
+const path = require('path')
+var assemblyscript
+(() => {
+  try {
+    assemblyscript = require('assemblyscript')
+  } catch (e) {
+    require('ts-node').register({ project: path.join(__dirname, 'node_modules', 'assemblyscript', 'src', 'tsconfig-base.json') })
+    require('./node_modules/assemblyscript/src/glue/js')
+    assemblyscript = require('./node_modules/assemblyscript/src')
+  }
+})()
+const {
+  CommonFlags,
+  Node,
+  NodeKind,
+  SourceKind,
+  TypeKind,
+  parseFile
+} = assemblyscript
+
+exports.afterParse = function(parser) {
+
+  const entrySrcIdx = parser.program.sources.findIndex(s => s.isEntry)
+  const entrySrc = parser.program.sources[entrySrcIdx]
+  
+  entrySrc.statements.forEach(stmt => {
+    //console.log(Object.keys(stmt))
+    if (
+      stmt.kind === NodeKind.FUNCTIONDECLARATION &&
+      stmt.decorators &&
+      stmt.decorators.length &&
+      stmt.decorators[0].name.text === "zome_function"
```

Comment:
I think better use `stmt.decorators.filter(d =>  d.name.text === "zome_function").length` because user could specified more than one of decorators with random order like
```ts
@inline
@zome_function
function debug(str: string) {
   ...
}
```